### DISCUSSION TITLE
Clarifying the Relationship Between Stellarium Mobile and Desktop Versions

### DISCUSSION THREAD

**Discussion by imreverse** [Stellarium/stellarium#3957] [General]
I recently learned that the Stellarium mobile app is not open-source, unlike the desktop version, which was developed by the same author.

What confuses me is that both websites—stellarium.org (the FOSS version) and stellarium-labs.com (the non-FOSS version?)—link to the same web version, stellarium-web.org (foss?). Does this imply that the two projects are somehow connected since they share a common web portal?

Also, if I pay for the mobile version, does any portion of the contribution go to support the open-source community?

**Comment by gzotti** [comment]
You find an open-source stellarium-web here. 
Paying the little amount for Stellarium mobile allows for some continued support also for the FOSS desktop version, yes.

**Comment by 10110111** [comment]
The web and mobile versions share (closed-source) code and originate from the same opensource project: [stellarium-web-engine](https://github.com/Stellarium/stellarium-web-engine), which hasn't been updated for a few years now.
